benchexec/tools/sbt-fizzer.py

```diff
@@ -0,0 +1,148 @@
+import benchexec.tools.template
+import benchexec.result as result
+from benchexec.tools.sv_benchmarks_util import get_data_model_from_task, ILP32, LP64
+import os
+
+class Tool(benchexec.tools.template.BaseTool2):
+    """
+    Tool info for sbt-fizzer.
```

Comment:
Is there more information that you could give about the tool here, like its homepage?